config/samples/targets/azuresentineltarget/200-target.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: targets.triggermesh.io/v1alpha1
+kind: AzureSentinelTarget
+metadata:
+  name: hello-sentinel
+spec:
+  # subscriptionID refers to the Azure Subscription ID that the Azure Sentinel instance is associated with.
```

Comment:
I'd remove these comments from here, they are already added in the CRD

Reply:
I prefer to be over-documented than under... I like having an explanation of the spec parameters in the samples. Not everyone knows what a CRD is or how to read them. But i will remove them if there is a general consensus here.